src/Tickets/Seating/Settings.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Seating settings.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Seating
+ */
+
+namespace TEC\Tickets\Seating;
+
+use TEC\Common\Contracts\Provider\Controller as Controller_Contract;
+use TEC\Common\StellarWP\Arrays\Arr;
+
+/**
+ * Class Settings.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Seating
+ */
+class Settings extends Controller_Contract {
+	/**
+	 * The option name for the frontend timer setting.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public const FRONTEND_TIMER_OPTION = 'ticket-seating-frontend-timer';
```

Comment:
Can we make the option name more obvious of what is it used for ?

e.g. `tickets-seatings-timer-duration`